### PR TITLE
*: update default upstream URL

### PR DESCRIPTION
**Pull request by crawford**
The production Cincinnati stack is going to run at the given URL. Once
it has been turned up, CVO will be able to automatically check for
updates.

cc @smarterclayton @steveej @jhernand

Hold while we wait for confirmation about that URL...
/hold

### CODE REVIEW

File: docs/dev/metrics.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ cluster_version{payload="test/image:1",type="current",version="4.0.2"} 1
 cluster_version{payload="test/image:1",type="failure",version="4.0.2"} 1
 # HELP cluster_version_available_updates Report the count of available versions for an upstream and channel.
 # TYPE cluster_version_available_updates gauge
-cluster_version_available_updates{channel="fast",upstream="http://localhost:8080/graph"} 0
+cluster_version_available_updates{channel="fast",upstream="https://api.openshift.com/api/upgrade_info/v1/graph"} 0
```
Comment (steveeJ) [inline docs/dev/metrics.md]: FWIW: Juan anticipated *upgrade_mgmt* as the prefix on the E-Mail thread on this topic. The previous anticipation was *update_mgmt*.

File: pkg/cvo/cvo.go [hunk 2]
```diff
@@ -144,7 +144,7 @@ func New(
 
 		minimumUpdateCheckInterval: minimumInterval,
 		payloadDir:                 overridePayloadDir,
-		defaultUpstreamServer:      "http://localhost:8080/graph",
+		defaultUpstreamServer:      "https://api.openshift.com/api/upgrade_info/v1/graph",
```
Comment (steveeJ) [inline pkg/cvo/cvo.go]: FWIW: Juan anticipated *upgrade_mgmt* as the prefix on the E-Mail thread on this topic. The previous anticipation was *update_mgmt*.

**Comment by jhernand** [comment]
@steveeJ in the mail thread that you mention we agreed to use `upgrades_info`, not the plural in _upgrades_.

**Comment by crawford** [comment]
@jhernand Ah, thanks for the catch. I've updated the PR.

**Comment by crawford** [comment]
> error: unable to connect to image repository registry.svc.ci.openshift.org/ci-op-zb792m7w/stable@sha256:6cd9842c7897cff8eb05d6e3a9b9dd05de21ae484a68d2b7dba8d71e58b7c22d: Get https://registry.svc.ci.openshift.org/v2/: net/http: request canceled (Client.Timeout exceeded while awaiting headers)

/retest

**Comment by crawford** [comment]
My previous `/hold` didn't take...

/hold

**Comment by smarterclayton** [comment]
Approved for beta, you may merge this when it's ready.

**Pull request merged**